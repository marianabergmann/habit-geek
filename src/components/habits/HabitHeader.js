import React from "react";

const HabitHeader = () => {
  return (
    <div className="habit__row">
      <div className="habit__cell-habit-container"></div>
      <div className="habit__cell-frequency-container">
        <h5 className="habit__cell">Mon</h5>
        <h5 className="habit__cell">Tue</h5>
        <h5 className="habit__cell">Wed</h5>
        <h5 className="habit__cell">Thu</h5>
        <h5 className="habit__cell">Fri</h5>
        <h5 className="habit__cell">Sat</h5>
        <h5 className="habit__cell">Sun</h5>
      </div>
    </div>
  );
};

export default HabitHeader;
