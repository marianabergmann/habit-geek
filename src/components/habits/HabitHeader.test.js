import React from "react";
import { shallow } from "enzyme";
import HabitHeader from "./HabitHeader";

describe("HabitHeader component", () => {
  it("Renders static elements", () => {
    const habitHeader = shallow(<HabitHeader />);
    expect(habitHeader.find(".habit__cell-action-container").length).toBe(1);
    expect(habitHeader).toMatchSnapshot();
  });

  it("Hides header placeholder when on readonly mode", () => {
    const habitHeader = shallow(<HabitHeader isReadOnly={true} />);
    expect(habitHeader.find(".habit__cell-action-container").length).toBe(0);
  });
});
