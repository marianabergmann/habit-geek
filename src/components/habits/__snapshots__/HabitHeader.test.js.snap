// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HabitHeader component Renders static elements 1`] = `
<div
  className="habit__row"
>
  <div
    className="habit__cell-habit-container"
  />
  <div
    className="habit__cell-frequency-container"
  >
    <h5
      className="habit__cell"
    >
      Mon
    </h5>
    <h5
      className="habit__cell"
    >
      Tue
    </h5>
    <h5
      className="habit__cell"
    >
      Wed
    </h5>
    <h5
      className="habit__cell"
    >
      Thu
    </h5>
    <h5
      className="habit__cell"
    >
      Fri
    </h5>
    <h5
      className="habit__cell"
    >
      Sat
    </h5>
    <h5
      className="habit__cell"
    >
      Sun
    </h5>
  </div>
  <div
    className="habit__cell-action-container"
  />
</div>
`;
